<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDataCreateSelectionFrame</class>
 <widget class="QWidget" name="FindDataCreateSelectionFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="selectionType">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>from</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="pqOutputPortComboBox" name="source">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="pqMultiQueryClauseWidget" name="queryClauseWidget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="runQuery">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Run Selection Query</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqOutputPortComboBox</class>
   <extends>QComboBox</extends>
   <header>pqOutputPortComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>pqMultiQueryClauseWidget</class>
   <extends>QWidget</extends>
   <header>pqMultiQueryClauseWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
